lib/ProductOpener/Products.pm

```diff
@@ -1452,11 +1452,11 @@ sub store_product ($user_id, $product_ref, $comment) {
 
 	$log->debug("store_product - done", {code => $code, product_id => $product_id}) if $log->is_debug();
 
-	# index for search service
-	push_to_search_service($product_ref);
+	my $update_type = $product_ref->{deleted} ? "deleted" : "updated";
+	# Publish information about update on Redis stream
+	push_to_redis_stream($user_id, $product_ref, $update_type, $comment, $diffs);
```

Comment:
I'm concerned that if we have bulk updates that the diff could bloat the stream size. Do we need to send the diffs to redis? I couldn't see that search-a-licious is using them?

Reply:
Search-a-licious doesn't require the diff, but Robotoff does. And we plan to switch Robotoff to Redis streams as well. It only contains the names of the fields that were updated/deleted/created though, not the content.

Reply:
OK, I wasn't fully aware of what the diff contained